src/applications/debt-letters/const/diary-codes/index.js

```diff
@@ -317,90 +237,75 @@ export const renderAdditionalInfo = (diaryCode, dateOfLetter) => {
-            <a href="#howDoIPay" className="vads-u-margin-y--2">
-              Review payment options
-            </a>
-          </p>
-        </div>
+        <p>
+          <strong>Next step: </strong>
+          We'll let you know when we start collecting on this debt again. You
+          don't have to do anything until that time. But you can choose to pay
+          part or all of the debt now if you'd like.{' '}
+          <a href="#howDoIPay" className="vads-u-margin-y--2">
+            Review payment options
+          </a>
+        </p>
       );
     case '080':
     case '850':
     case '852':
     case '860':
     case '855':
       return (
-        <div>
-          <p>
-            <strong>Status: </strong>
-            We referred this debt to the U.S. Department of the Treasury.
-          </p>
-          <p>
-            <strong>Next step: </strong>
-            Call the U.S. Department of the Treasury's Debt Management Center at
-            888-826-3127, 8:30 a.m. to 6:30 p.m. ET. Don't send us payment
-            directly. This will delay posting of payment to your account. And
-            the Treasury Department may continue adding fees and interest.
-          </p>
-        </div>
+        <p>
+          <strong>Next step: </strong>
+          Call the U.S. Department of the Treasury's Debt Management Center at
+          888-826-3127, 8:30 a.m. to 6:30 p.m. ET. Don't send us payment
+          directly. This will delay posting of payment to your account. And the
+          Treasury Department may continue adding fees and interest.
+        </p>
       );
     case '081':
     case '500':
     case '510':
     case '503':
       return (
-        <div>
-          <p>
-            <strong>Status: </strong>
-            We're referring this debt to the U.S. Department of the Treasury
-            today.
-          </p>
+        <>
           <p>
             <strong>Next step: </strong>
             Please pay the full amount online or by phone now. If we don't
-            receive your payment by [date/time], we're required by law to refer
-            your debt to the U.S. Department of the Treasury. After we do that,
-            we can't help you manage your debt.
+            receive your payment by <strong>[date/time]</strong>, we're required
+            by law to refer your debt to the U.S. Department of the Treasury.
+            After we do that, we can't help you manage your debt.
           </p>
           <p>
-            <a href="https://www.pay.va.gov/">Pay online now</a> Or may a
-            payment by phone at 800-827-0648 (or 1-612-713-6415 from overseas).
-            We’re here Monday through Friday, 7:30 a.m. to 7:00 p.m. ET.
+            <a href="https://www.pay.va.gov/">Pay online now</a> Or{' '}
+            <strong>make</strong> a payment <strong>over the</strong> phone at
+            <ContactDMC />. We’re here Monday through Friday, 7:30 a.m. to 7:00
+            p.m. ET.
```

Comment:
It might make life easier if the `. We’re here Monday through Friday, 7:30 a.m. to 7:00 p.m. ET.` was included inside the `<ContactDMC />` component. 😀

Reply:
yea unfortunately the content is different for certain cases which prevented me from adding that part